src/Template/AddParametersTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Template;
+
+trait AddParametersTrait
+{
+    /**
+     * @var array
+     */
+    protected $templateParams = [];
```

Comment:
Since these members (this and the `mergeParameters()` method) are in the trait, they can be `private` visibility.